charts/llm-d-modelservice/templates/epp-service.yaml

```diff
@@ -1,17 +1,17 @@
-{{- if .Values.endpointPicker -}}
+{{- if .Values.routing.epp.create -}}
 apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "llm-d-modelservice.fullname" . }}-epp-service
   labels:
     {{- include "llm-d-modelservice.labels" . | nindent 4 }}
 spec:
-  type: {{ .Values.endpointPicker.service.type }}
+  type: {{ default "ClusterIP" .Values.routing.epp.service.type }}
```

Comment:
Again I think the default choice should be set in the values or values + helpers if you need it